src/Middleware/Middleware.slnf

```diff
@@ -125,6 +125,8 @@
       "src\\Servers\\Kestrel\\Kestrel\\src\\Microsoft.AspNetCore.Server.Kestrel.csproj",
       "src\\Servers\\Kestrel\\Transport.Quic\\src\\Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.csproj",
       "src\\Servers\\Kestrel\\Transport.Sockets\\src\\Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.csproj",
+      "src\\Testing\\src\\Microsoft.AspNetCore.Testing.csproj",
+      "src\\Testing\\test\\Microsoft.AspNetCore.Testing.Tests.csproj",
```

Comment:
When I was settuping my dev environment with Visual Studio, I had a lot of issues with unresolved dependencies. After adding this two projects to filter, the issue disappeared. I don't know it that right thing to commit thought

Reply:
Adding Testing is fine. Testing.Tests shouldn't be needed, nothing should depend on it.

VS has a feature where you can right-click on a project and tell it to load all transitive dependencies.

Reply:
Agreed, please remove Microsoft.AspNetCore.Testing.Tests.csproj